Sources/App/CRP/CRPProcess.swift

```diff
@@ -2,12 +2,14 @@ import Foundation
 import Vapor
 import Stevenson
 
+/**
+ For detailed documentation of this part of the code, see: [Implementation Details documentation in private repo](https://github.com/babylonhealth/babylon-ios/blob/develop/Documentation/Process/Release%20process/CRP-Bot-ImplementationDetails.md#executing-the-crp-process)
+*/
 enum CRPProcess {
     enum Option: String {
         case repo
         case branch
-        case skipTicket
-        case skipFixVersion
+        case slack_channel_id
```

Comment:
Looks like other types are using camel case convention:
```suggestion
        case slackChannelID
```